### PR TITLE
Run buildifier --lint=fix

### PR DESCRIPTION
**Pull request by laurentlb**
This fixes multiple issues, related to future changes in Bazel.

Remaining work: make the code compatible with `--incompatible_depset_is_not_iterable` and `--incompatible_depset_union`.

### CODE REVIEW

File: closure/filegroup_external.bzl [hunk 1]
```diff
@@ -28,11 +28,11 @@ def _filegroup_external(ctx):
         basename = ""
         for url in urls:
             basename = url[url.rindex("/") + 1:] or basename
-            if url in downloaded:
+            if url in downloaded.to_list():
```
Comment (gkdn) [inline closure/filegroup_external.bzl]: Isn't there a more efficient way to do contains check in depssets?

File: closure/private/defs.bzl [hunk 2]
```diff
@@ -188,7 +188,7 @@ def find_js_module_roots(srcs, workspace_name, label, includes):
                         break
                 if not found:
                     fail("No srcs found beginning with '%s/'" % include)
-            for root in roots:
+            for root in roots.to_list():
```
Comment (gkdn) [inline closure/private/defs.bzl]: is to_list cached if called multiple times on a depset?

**Comment by laurentlb** [comment]
The rule isn't using depsets in an optimal way. This is why it doesn't work with `--incompatible_depset_is_not_iterable` and `--incompatible_depset_union` - and we should fix that.

The calls to `to_list` were inserted automatically by `buildifier --lint=fix`. It simply makes more explicit what was happening behind the scene.

I can either leave the code like this, or revert the two lines you pointed out. In both cases, this should ideally be revisited in the future.

**Comment by laurentlb** [comment]
Please merge it (I don't have write access)

**Pull request merged**